Add `History of osu! 2023` (#12027)
ppy/osu-wiki: wiki/History_of_osu!/2023/en.md

# 2023

## January

On the second week of January, [peppy](https://osu.ppy.sh/users/2) was interviewed by [MongoDB](https://en.wikipedia.org/wiki/MongoDB), the team responsible for developing [Realm](https://realm.io/), a database system used in osu!(lazer). In the interview, they talked about topics such as the origin of osu!, how osu! is making revenue, osu!'s development and infrastructure as well as the transition to osu!(lazer).[^interview-mongodb]

Due to an update to osu!(lazer) on 21 January, mappers became able to set the preview time of a beatmap in the [beatmap editor](/wiki/Client/Beatmap_editor). Furthermore, the gameplay progress bar was redesigned for the "argon" skin and the ability to change the screen's coverage of the [Fade In](/wiki/Gameplay/Game_modifier/Fade_In) mod was added. Lastly, [performance points](/wiki/Performance_points) were enabled for scores which caused a spike in player activity. Right after the update went live, the scores set per day increased from 103,000 the day earlier to 302,000 and the concurrent player count rose from around 200 to 1,600.[^lazer-jan-21]

> Thanks to [ThePoon](https://osu.ppy.sh/users/718454) and other contributors, [osu!stream](/wiki/osu!stream) was ported to [Android](https://en.wikipedia.org/wiki/Android_(operating_system)), a game specifically designed for handheld devices and with unique gameplay elements. Until then, it was only available on [iOS](https://en.wikipedia.org/wiki/IOS). An implementation on Android was not planned at the time when the development of the iOS version began in 2011 due to audio-related restrictions in Android's infrastructure that would heavily affect the gameplay experience.[^osu-stream-twitter][^osu-stream-pr]

Reviewer: ```suggestion
Thanks to [ThePooN](https://osu.ppy.sh/users/718454) and other contributors, [osu!stream](/wiki/osu!stream) was ported to [Android](https://en.wikipedia.org/wiki/Android_(operating_system)), a game specifically designed for handheld devices and with unique gameplay elements. Until then, it was only available on [iOS](https://en.wikipedia.org/wiki/IOS). An implementation on Android was not planned at the time when the development of the iOS version began in 2011 due to audio-related restrictions in Android's infrastructure that would heavily affect the gameplay experience.[^osu-stream-twitter][^osu-stream-pr]
```

Reviewer: Yeah nvm u can ignore this, Wala did some changes

> Thanks to [ThePoon](https://osu.ppy.sh/users/718454) and other contributors, [osu!stream](/wiki/osu!stream) was ported to [Android](https://en.wikipedia.org/wiki/Android_(operating_system)), a game specifically designed for handheld devices and with unique gameplay elements. Until then, it was only available on [iOS](https://en.wikipedia.org/wiki/IOS). An implementation on Android was not planned at the time when the development of the iOS version began in 2011 due to audio-related restrictions in Android's infrastructure that would heavily affect the gameplay experience.[^osu-stream-twitter][^osu-stream-pr]

Reviewer: Should be capitalized with `ThePooN` btw

## August

From May through August, the [Icons of an Era](https://osu.ppy.sh/home/news/2023-05-10-skinning-contest-icons-of-an-era) skinning contest was organised to celebrate 15 years of osu! in the form of [skins](/wiki/Skin) representing popular beatmaps or mascots. [thetasigma](https://osu.ppy.sh/users/6234482), [Chromasia](https://osu.ppy.sh/users/7306251), and [Spoo](https://osu.ppy.sh/users/11805037) won the third, second, and first place in the contest respectively. The prize included a custom badge and varying amounts of osu!supporter.[^icons-of-an-era]

[Dynam1cNET](https://osu.ppy.sh/users/12540789) wanted to show what is possible with the me! page and implemented a custom version of r/place, an event on [Reddit](https://www.reddit.com) where users were able to *place* coloured pixels on a canvas.[^me-page-place]

## November

Several people were commended and awarded with a Community Contributor badge for their remarkable assistance in the osu! community in 2022 and 2023.[^contributors]

## December

> In parallel to this year's last update, the osu! team wanted more people to try out osu!(lazer) and give feedback via the forum before locking gameplay mechanics and adding the pp system.[^feedback] Gaining a miss caused less penalty for high combos (`combo^1.5` compared to `combo^2`). The combo–accuracy split was rebalanced, making accuracy account for a bigger portion in a score. osu!catch's grading in osu!(lazer) was changed to match the grading of osu!(stable) to simplify obtaining high accuracy values. At last, sliders became more lenient.[^lazer-12-18]

Reviewer 2: ```suggestion
In parallel to this year's last update, the osu! team wanted more people to try out osu!(lazer) and give feedback via the forum before finalising gameplay mechanics and allowing [performance points](/wiki/Performance_points) to be awarded for scores.[^feedback] The following notable changes happened:

- The score penalty for misses at high combos was decreased (`combo^1.5` compared to `combo^2`).
- The combo–accuracy split was rebalanced, making accuracy account for a bigger portion in a score.
- osu!catch's grading in osu!(lazer) was changed to match the grading of osu!(stable) to simplify obtaining high accuracy values.
- Slider mechanics were adjusted to make fast sliders more lenient.[^lazer-12-18]
```

> The 2024's [Winter Fanart Contest](https://osu.ppy.sh/home/news/2023-11-07-winter-fanart-contest) ran under the theme "osu! mascots enjoying a seasonal feast". Due to this broad theme, the participants had a lot of freedom and creativity for designs. All finalists obtained three months of osu!supporter, while the top 3 winners received a custom [profile badge](/wiki/Community/Profile_badge) on top.[^winter-fanart]

Reviewer 2: ```suggestion
2024's [Winter Fanart Contest](https://osu.ppy.sh/home/news/2023-11-07-winter-fanart-contest) ran under the theme "osu! mascots enjoying a seasonal feast". Due to this broad theme, the participants had a lot of freedom and creativity for designs. All finalists obtained three months of osu!supporter, while the top 3 winners received a custom [profile badge](/wiki/Community/Profile_badge) on top.[^winter-fanart]
```
